gbpservice/nfp/configurator/drivers/loadbalancer/v1/haproxy/haproxy_lb_driver.py

```diff
@@ -36,7 +40,128 @@
 LOG = logging.getLogger(__name__)
 
 
-class HaproxyOnVmDriver(base_driver.BaseDriver):
+""" Loadbalancer generic configuration driver for handling device
+configuration requests.
+
+"""
+
+
+class LbGenericConfigDriver(object):
+    """
+    Driver class for implementing loadbalancer configuration
+    requests from Orchestrator.
+    """
+
+    def __init__(self):
+        self.timeout = 30
+
+    def _configure_log_forwarding(self, url, mgmt_ip, port):
```

Comment:
This log forwarding function looks to be same in both FW and LB. Should we put this in base driver?